### ISSUE TITLE
Related WordPress objects -> Salesforce sequences

### ISSUE THREAD

**Issue by jonathanstegall** [MinnPost/object-sync-for-salesforce#171]
### Expected behavior

I think once we're dealing with bigger systems like WooCommerce, users expect that they should be able to take a group of related data - a voucher, or an order, or something in WooCommerce that is also tied to a customer, a user, whatever other related data - and use it to create multiple things in Salesforce - a Contact and an Opportunity, or some other combination.

Part of the trickiness of this is that we can't anticipate all the possible data storage methods. WooCommerce uses many custom tables, and I think any system like it would also use custom tables.

### Actual behavior

We have no direct way to facilitate this right now, because we can only map single objects to single objects. Certainly it appears to be possible to achieve it with developer hooks, but it is rather complicated and it is no longer based in the plugin interface ([example here](https://wordpress.org/support/topic/relationship-between-sf-objects/#post-10235489))

### Steps to reproduce behavior

Basically try to map literally anything in WooCommerce.

**Comment by jonathanstegall** [comment]
Some forum questions about doing this specifically with taxonomies:

- https://wordpress.org/support/topic/mapping-to-categories-from-salesforce/
- https://wordpress.org/support/topic/how-to-map-salesforce-drop-down-list-to-custom-taxonomy-in-wordpress/
